app/src/main/java/com/example/antonynganga/android_freckle/services/AuthService.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.antonynganga.android_freckle.services
+
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface AuthService {
+
+    @GET("/")
+    fun getIndexUrl(): Call<String>
```

Comment:
Used a generic endpoint. Auth endpoints should be specified later on in development.